Documentation/specs/runtime-configuration-file.md

```diff
@@ -2,206 +2,155 @@
 
 The runtime configuration files store the dependencies of an application (formerly stored in the `.deps` file). They also include runtime configuration options, such as the Garbage Collector mode. Optionally they can also include data for runtime compilation (compilation settings used to compile the original application, and reference assemblies used by the application).
 
-**Note:** This document doesn't provide full explanations as to why individual items are needed in this file. That is covered in the [`corehost` spec](corehost.md) and via the `Microsoft.Extensions.DependencyModel` assembly.
+**Note:** This document doesn't provide full explanations as to why individual items are needed in this file. That is covered in the [`apphost` spec](corehost.md) and via the `Microsoft.Extensions.DependencyModel` assembly.
```

Comment:
Aside: is this the best place for this document? Should it go in core-setup? Or docs.microsoft.com?

Reply:
Yes I could see the "corehost" spec be moved to core-setup documentation to be consistent with other related documents also located there.